### PR TITLE
14022 create adjust bash script to to configure nginx and public html for geocint dashboard

### PR DESCRIPTION
**Pull request by Zkir**

### CODE REVIEW

File: scripts/Makefile [hunk 1]
```diff
@@ -90,8 +80,42 @@ ${GEOCINT_WORK_DIRECTORY}/geocint/installation/database_block_finished: ${GEOCIN
 	echo "Database initialisation step finished successfully"
 	touch $@
 
+# ------------------------ WEB Dashboard installation block ----------------------------
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/nginx_installation: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished  ## install and configure nginx
+	sudo systemctl stop apache2
```
Comment (Komzpa) [inline scripts/Makefile:85]: How does apache appear on the system and why are we in the right to disable it silently?

File: scripts/Makefile [hunk 2]
```diff
@@ -90,8 +80,42 @@ ${GEOCINT_WORK_DIRECTORY}/geocint/installation/database_block_finished: ${GEOCIN
...
+	sudo systemctl restart nginx
+	touch $@
+
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/set_modes: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished  ## create public_html folder and set access
+	sudo usermod -aG adm ${USER}
+	sudo usermod -aG ${USER} www-data
+	sudo mkdir -p ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chown ${USER}:${USER} ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo chmod 0750 ${GEOCINT_WORK_DIRECTORY}/public_html
+	sudo echo 'export PATH=$$PATH:/usr/local/pgsql/bin' >> ${HOME}/.bashrc
```
Comment (Komzpa) [inline scripts/Makefile:101]: why is this necessary on packaged postgres?

**Review by Komzpa — CHANGES_REQUESTED** [review]

File: scripts/Makefile (hunk 2 above)
Comment (frolui) [reply scripts/Makefile:101]: you are right, it's redundant

Comment (frolui) [reply scripts/Makefile:101]: @Zkir can you remove this line?

Comment (Zkir) [reply scripts/Makefile]: Removed.

File: scripts/Makefile (hunk 1 above)
Comment (Zkir) [reply scripts/Makefile]: Apache is coming by default with Ubuntu 22.04 LTS.

I was going to make complete automatic install, including creation of nginx configurations. 
However, automatic nginx configuration turned out to be not feasible (for me),  so  currently it all is moved to manual steps and described in DOCUMENTATION.md

**Review by frolui — APPROVED** [review]

**Comment by frolui** [comment]
Tested on VirtualBox Ubuntu installation (passed successfully)

**Review by Komzpa — DISMISSED** [review]

**Review by frolui — APPROVED** [review]
